features/election.feature

```diff
@@ -21,6 +21,22 @@ Feature: Election
     When I vote on an election
     Then I see alert "Takk for din stemme!"
 
+  Scenario: Retrieve vote with valid hash
+    Given There is an active election
+    And I have voted on the election
+    And I am on the retrieve vote page
+    When I submit the form
+    Then I see "Kvittering:"
```

Comment:
Don't think there's any point in checking the titles, should just verify that the actual values are correct. Could use `element(by.binding('vote.fieldGoesHere')` to match `{{vote.fieldGoesHere}}`.